SoloProject/basic-tags.html

```diff
@@ -86,19 +86,24 @@ <h2>Lists!</h2>
       <p>There are two different types of lists in html, ordered and unordered lists. Unordered lists start with bullets, and ordered start with numbers.</p>
     </section>
     <section>
-      <h2 id="tags">Different Flowers</h2>
-      <ul>
-        <li>Lily</li>
-        <li>Aster</li>
-        <li>Tulip</li>
-        <li>Jasmine</li>
-        <li>Poppy</li>
-        <li>Lilac</li>
-        <li>Peony</li>
-      </ul>
+      <h2 id="flowers">Different Flowers</h2>
+      <section id="eleven">
+        <h2 id="flora">Different Flowers</h2>
+        <ul>
+          <li>White Flowers
+          <ul>
+            <li>Lily</li>
+            <li>Daisy</li>
+            <li>Baby's Breath</li>
+          </ul>
+          </li>
+        <li> Blue Flowers</li>
+    </ul>
```

Comment:
Be careful with indentation; it's impossible for me to tell by sight whether the tags are properly nested here. Even if I read the code _verrrrrrrrry_ carefully, I'm not going to trust that I read it faithfully.

The same is going to be true for you as you write code. VS Code will indent things the "right way" by default. You can also install VS Code extensions that will properly indent your code after the fact.

Ideally, though, you're doing it as you go so that the indentation serves as a sanity check _for yourself_.